omod/src/main/webapp/fragments/managePreferences.gsp

```diff
@@ -178,41 +178,41 @@
-	<label><input type="checkbox" class="enable-disable-check" id="enable_duplicateresults" />Include Duplicate Results</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_history" />${ ui.message("chartsearch.refApp.manage.preferences.enableHistory") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_bookmarks" />${ ui.message("chartsearch.refApp.manage.preferences.enableBookmarks") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_notes" />${ ui.message("chartsearch.refApp.manage.preferences.enableNotes") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_quicksearches" />${ ui.message("chartsearch.refApp.manage.preferences.enableQuickSearches") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_defaultsearch" />${ ui.message("chartsearch.refApp.manage.preferences.enableDefaultSearch") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_multiplefiltering" />${ ui.message("chartsearch.refApp.manage.preferences.enableMultiFilterings") }</label><br />
+	<label><input type="checkbox" class="enable-disable-check" id="enable_duplicateresults" />${ ui.message("chartsearch.refApp.manage.preferences.enableDuplicates") }</label><br />
 	<br />
 </div>
 
-<div class="pref-summary-item" id="pref-summary-item-categoryfilter">Category Filters</div>
+<div class="pref-summary-item" id="pref-summary-item-categoryfilter"> ${ ui.message("chartsearch.refApp.manage.preferences.categoryFilters") }</div>
 <div id="pref-summary-item-categoryfilter-details" class="pref-summary-item-details">
 	<br />
-		<b>NOTE:</b> Category Names are displayed by default, providing your preferred Display name will instead display your name than the default<br /><br />
+		<b>${ ui.message("chartsearch.note") }</b> ${ ui.message("chartsearch.refApp.manage.preferences.categoryNote") }<br /><br />
 		<table id="preferences-cats"></table>
 	<br /><br />
 </div>
 
-<div class="pref-summary-item" id="pref-summary-item-notescolors">Notes Background Colors</div>
+<div class="pref-summary-item" id="pref-summary-item-notescolors">${ ui.message("chartsearch.refApp.manage.preferences.notesColor") }</div>
 <div id="pref-summary-item-notescolors-details" class="pref-summary-item-details">
 	<br />
-	<div><label for="color1">Click and Select color to add</label>
+	<div><label for="color1">${ ui.message("chartsearch.refApp.manage.preferences.addColor") }</label>
 		<input id="color1" name="color1" type="text" value="#333399" />
 		<select id="choose-color-collection">
-				<option>Color Collection To choose from</option>
-				<option>Default Colors</option>
-				<option>Red-based Colors</option>
-				<option>Green-based Colors</option>
-				<option>Blue-based Colors</option>
+				<option>${ ui.message("chartsearch.refApp.manage.preferences.notesColor") }</option>
```

Comment:
Actually go ahead and push all other changes you are making to your branch from which you have created the pull request and such commits will automatically be added into the Pull Request